src/libsync/account.h

```diff
@@ -111,6 +111,9 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
     /** Returns webdav entry URL, based on url() */
     QUrl davUrl() const;
 
+    /** Returns zsync metadata URL, based on url() */
+    QUrl zsyncUrl() const;
```

Comment:
See my comments elsewhere about having the zsync metadata file URL as PROPFIND property instead